amplify/team-provider-info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dev": {
+        "awscloudformation": {
+            "AuthRoleName": "rickyism-generator-dev-20190615123919-authRole",
+            "UnauthRoleArn": "arn:aws:iam::106553238186:role/rickyism-generator-dev-20190615123919-unauthRole",
```

Comment:
These values should be considered secret. Can we change these to place holders (with secrets saved in Parameter Store), and update history so these values can't be noticed?

Or could take the repo private